test/create-branch/in/.keboola/manifest.json

```diff
@@ -4,7 +4,7 @@
     "id": %%TEST_KBC_PROJECT_ID%%,
     "apiHost": "%%TEST_KBC_STORAGE_API_HOST%%"
   },
-  "sortBy": "id",
+  "sortBy": "path",
```

Comment:
Zistil som, ze v niektorych testoch bolo nastavene blbe poradie, ... zaujimave, ze to zacalo nahodne padat az teraz.